test/test_command.py

```diff
@@ -66,3 +66,96 @@ def test_parse_commands_failure(capsys, command, sys_exit_code, sys_exit_message
 
     captured_stderr = capsys.readouterr().err
     assert sys_exit_message in captured_stderr
+
+
+TEST_ARGUMENT_T_INPUT = """otk.version: "1"
+
+otk.target.osbuild.name:
+  pipelines:
+    - "test"
+"""
+
+TEST_ARGUMENT_T_INPUT_TWO_TARGETS = """otk.version: "1"
```

Comment:
Could this now be written as:
```python
TEST_ARGUMENT_T_INPUT_TWO_TARGETS = TEST_ARGUMENT_T_INPUT  + """
otk.target.osbuild.name1:
  pipelines:
    - "test1"
 """"
 ?
```

Reply:
(obviously a nitpick but I really like to avoid g duplication (within reason) :)